Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM golang:1.20-alpine as builder
```

Comment:
could we use just `golang:1.20`? I ask because we already use that in other places (like https://github.com/buildpacks/github-actions) and I'd prefer not to worry about additional OSes

Reply:
The only reason I used alpine was to get the ca-certificates into the image in order to have parity with the current image (which installs them with the ca-certificates buildpack). The base image will actually be scratch. I’m happy to use a different golang image but I think we need to install ca-certificates regardless so it seemed like the easiest approach.